chingu-voyage4/Geckos-Team-32
# Pages/landing

## Purpose
* Re-factor front-end to handle youtube request

## Approach/Fixes
* Re-factor file setup
* Display youtube results on post-landing page

## Testing
* Download this PR to your local machine
* Add youtube API key to line 21 of AppRoutes.jsx
* run `npm run dev`
* Type in a sear query
* Hit the button to the post-landing page and check that the results show
* Re-type a new query on the post landing site and see that the page updates
